### ISSUE TITLE
Is there a way to share compiled packages?

### ISSUE THREAD

**Issue by beroal** [vn971/rua#157]
I see that compiled packages are put deep inside the home directory of the user running `rua`. In order to share compiled packages, I execute  `mount --bind /home/${USER}/.local/share/rua/checked_tars /usr/share/rua/checked_tars`. Is there a better way, probably, a command-line option for `rua` for setting up this directory?

**Comment by vn971** [comment]
@beroal you mean that you'd like RUA to put built packages in /usr/share/... ?

To clarify a bit on how rua works: it follows the XDG specification when it can. `checked_tars` in particular are stored in the "data directory", used via this library function: https://github.com/dirs-dev/directories-rs/blob/main/src/lib.rs#L206

What would also work in practice is to override `XDG_DATA_HOME` environment variable (rua doesn't store anything else in XDG_DATA_HOME anyway).

Does that answer the concern?

**Comment by vn971** [comment]
@beroal I've improved the docs for that additionally: https://github.com/vn971/rua/commit/f2430057255b8a39148b02998452af5a112cc739

**Comment by beroal** [comment]
> Does that answer the concern?

Yes. If I set appropriate permissions for `/usr/share/rua` and start `rua` with the environment variable `XDG_DATA_HOME` set to `/usr/share`, `rua` puts compiled packages directly into `/usr/share/rua/checked_tars`. Thank you.

Actually, I knew these variables and these paths, but didn't realize you actually read the variables. You could have just put environment variable names into the table and say that they have default values according to the FreeDesktop specification.

**Comment by vn971** [comment]
Thanks for confirming!
I'll close the issue. With the documentation added to the bottom, I hope most people would grasp what's going on. If that'll prove to not be the case (e.g. raised issues), we could re-visit.
